app/connector/support/processor/src/test/java/io/syndesis/connector/support/processor/SyndesisHeaderStrategyTest.java

```diff
@@ -15,26 +15,31 @@
  */
 package io.syndesis.connector.support.processor;
 
+import org.apache.camel.CamelContext;
 import org.apache.camel.Exchange;
+import org.apache.camel.impl.DefaultExchange;
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
 
 public class SyndesisHeaderStrategyTest {
 
-    private static final Exchange NOT_USED = null;
+    CamelContext camelContext = mock(CamelContext.class);
+
+    Exchange exchange = new DefaultExchange(camelContext);
 
     @Test
     public void shouldFilterOutEverythingButInwardContentTypeHeader() {
         final SyndesisHeaderStrategy headerStrategy = new SyndesisHeaderStrategy();
 
-        assertThat(headerStrategy.applyFilterToCamelHeaders("Content-Type", "", NOT_USED)).isTrue();
-        assertThat(headerStrategy.applyFilterToExternalHeaders("Content-Type", "", NOT_USED)).isFalse();
+        assertThat(headerStrategy.applyFilterToCamelHeaders("Content-Type", "", exchange)).isTrue();
+        assertThat(headerStrategy.applyFilterToExternalHeaders("Content-Type", "", exchange)).isFalse();
```

Comment:
Was this needed?

Reply:
You wrote the test, I just made it pass :)